### PR TITLE
percentile

### PR DESCRIPTION
**Pull request by sgeorgevv**
Added percentile functionality #21386 across the board and fixed "check-formatting" issues in previous commit (#21256)

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by sgeorgevv** [comment]
hi @Infrared1029 , did you get a chance to take a look at this?

**Comment by Infrared1029** [comment]
Hi @sgeorgevv, there are several issues, ill try to point most of them in some comments, for some reason the CI is not triggered here.

File: ivy/functional/backends/jax/statistical.py [hunk 1]
```diff
@@ -215,3 +215,20 @@ def einsum(
...
+    /,
+    *,
+    q: Union[float, Sequence[float]],
+    axis: Optional[Union[int, Sequence[int]]] = None,
+    method: str = 'linear',
+    keepdims: bool = False,
+    out: Optional[JaxArray] = None,
+) -> Union[float, JaxArray]:
+    result = jnp.percentile(input, q, axis=axis, method=method, keepdims=keepdims)
+    if out is not None:
```
Comment (Infrared1029) [inline ivy/functional/backends/jax/statistical.py:230]: out arg is not supposed to be handled manually, you can remove these lines

File: ivy/functional/backends/tensorflow/statistical.py [hunk 2]
```diff
@@ -204,3 +205,20 @@ def einsum(
...
+    /,
+    *,
+    q: Union[float, Sequence[float]],
+    axis: Optional[Union[int, Sequence[int]]] = None,
+    method: str = 'linear',
+    keepdims: bool = False,
+    out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+) -> Union[tf.Tensor, tf.Variable]:
+    result = stats.percentile(input, q, axis=axis, interpolation=method, keepdims=keepdims)
+    if out is not None:
```
Comment (Infrared1029) [inline ivy/functional/backends/tensorflow/statistical.py:220]: same as jax's comment about `out`

File: ivy/functional/backends/torch/statistical.py [hunk 3]
```diff
@@ -320,3 +320,15 @@ def einsum(
...
+    input: torch.Tensor,
+    /,
+    *,
+    q: torch.Tensor,
+    axis: Optional[Union[int, Sequence[int]]] = None,
+    method: str = 'linear',
+    keepdims: bool = False,
+    out: Optional[torch.Tensor] = None,
+) -> torch.Tensor:
+    return torch.quantile(input, q, dim=axis, keepdim=keepdims, interpolation=method, out=out)
```
Comment (Infrared1029) [inline ivy/functional/backends/torch/statistical.py:334]: `quantile` returns outputs in the range [0, 1], `percentile` returns outputs in the range of [0, 100]

File: ivy/functional/ivy/experimental/activations.py [hunk 4]
```diff
@@ -458,3 +458,35 @@ def elu(
     }
     """
     return current_backend(x).elu(x, alpha=alpha, out=out)
+
+def sequence_length(
```
Comment (Infrared1029) [inline ivy/functional/ivy/experimental/activations.py:462]: not sure why this is added

**Comment by Infrared1029** [comment]
For some reason the CI is not being triggered, there might be other issues besides the ones I pointed out once the tests run and I can see the output, make sure that the tests are passing on your side as well.

File: ivy/functional/ivy/experimental/activations.py (hunk 4 above)
Comment (sgeorgevv) [reply ivy/functional/ivy/experimental/activations.py:462]: > not sure why this is added
@Infrared1029 
This was an edit to an older commit, I'm not sure why it merged both my pull requests into one.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
